corehq/apps/cloudcare/utils.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+
+
+def get_formplayer_url(domain, username):
+    return settings.FORMPLAYER_URL.format(
```

Comment:
this url locally will just be `localhost:8080`, so the format will do nothing. when it's in prod the url will be `"/formplayer/{domain}/{username}"`